index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NoteBook</title>
+    <style>
+        *{
+            margin: 0;
+            padding:0;
+            box-sizing: border-box;
+        }
+
+        body{
+            min-height: 100vh;
+            background:#d3d3d3;
+            font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+            display:flex;
+            justify-content:center;
+            align-items:center;
+        }
+        ::-webkit-scrollbar{
```

Comment:
is this mandatory ???

Reply:
Not mandatory its totally depend on ourself. I have used this because there was scroll bar in right side and it was not looking  that much good.